CHANGELOG.txt

```diff
@@ -1,5 +1,11 @@
-4.0.2
+4.1.0
+* Exposes billing metrics
```

Comment:
Is there an example demonstrating how to do this?

Reply:
yes, I've added examples into readme for java and scala:
- https://github.com/fauna/faunadb-jvm#per-query-metrics
- https://github.com/fauna/faunadb-jvm#per-query-metrics-1